README.md

```diff
@@ -66,9 +67,7 @@ You will need the following things properly installed on your computer.
 
 * [Git][git]
 * [Node.js][node]
-  * Supported versions:`v12`, `v14` and `v16`.
-  * **Apple Silicon Users**:  if you experience problems building or running
-    Boundary UIs, try Node v14.
+  * Supported versions:`v14`, `v16` and `v18`.
```

Comment:
Should we remove 14 support since it's already end of life? (Even 16 is losing security support in a month)

Reply:
@ZedLi That is true...  should we aim for `v16`, `v18` & `v20`? I will suggest to keep `v16` for a while (a month or two) since we just transition. What do you think?

Reply:
Yup that sounds great as long as `v20` also works!!

Reply:
@ZedLi Unfortunately, node 20 created a failure on electron-forge. After `0.14` release, we can take a look, hopefully `electron-forge` fixes the issue in upcoming version.

So for now node `v16` and `v18` support only.